README.rst

```diff
@@ -2,12 +2,87 @@
 
 .. |ImageLink| image:: https://travis-ci.org/bpython/bpython.svg?branch=master
 .. _ImageLink: https://travis-ci.org/bpython/bpython
-bpython - A fancy curses interface to the Python interactive interpreter
-========================================================================
 
+***********************************************************************
+bpython: A fancy curses interface to the Python interactive interpreter
+***********************************************************************
+
+`bpython`_ is a lightweight Python interpreter that adds several features common
+to IDEs. These features include **syntax highlighting**, **expected parameter
+list**, **auto-indentation**, and **autocompletion**. (See below for example
+usage).
+
+.. image:: http://i.imgur.com/jf8mCtP.gif
+  :alt: bpython
+  :width: 646
+  :height: 300
+  :align: center
+
+bpython does **not** aim to be a complete IDE - the focus is on implementing a
+few ideas in a practical, useful, and lightweight manner.
+
+bpython is a great replacement to any occassion where you would normally use the
+vanilla Python interpreter - testing out solutions to people's problems on IRC,
+quickly testing a method of doing something without creating a temporary file,
+etc..
+
+You can find more about bpython - including `full documentation`_ - at our
+`homepage`_.
+
+.. contents::
+  :local:
+    :depth: 1
+    :backlinks: none
+
+==========================
+Installation & Basic Usage
+==========================
+If you have `pip`_ installed, you can simply run:
+
+.. code-block:: bash
+
+    $ sudo pip install bpython
```

Comment:
`sudo` is problematic here. See http://notes.pault.ag/debian-python/ for some background. But in short: `sudo pip install` will break horribly if you also install Python modules with a package manager.